README.md

```diff
@@ -1,2 +1,89 @@
-# Space-Travelers-Hub
-React capstone project.
+# Space Travelers' Hub
+In this task, we worked with the real live data from the SpaceX API. Our task is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## Built With
+
+- Visual Studio Code and GitHub.
+- HTML, CSS and JavaScript.
+- Linters for HTML, CSS, JavaScript, React and Redux.
+- Gitflow.
+
+## Getting Started
+
+The Space Travelers' Hub consists of Rockets, Missions, and the My Profile section.
+
+# Rockets/Dragons
+
+The Rockets section displays a list of all available SpaceX rockets. Users can book each rocket by clicking the reservation button or cancel the previously made booking.
+
+# Missions
+
+The Missions section displays a list of current missions along with their brief description and participation status. There is also a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier.
+
+# My Profile
+
+The My Profile section displays all reserved rockets and space missions.
+
+## Live Demo
+
+[Space Travelers Hub](#)<br>
```

Comment:
- [ ] **[OPTIONAL]** Your live demo shows the repository instead of the deployed application. I suggest you deploy your app and add the link to the deployed app here instead of having the wrong link.